downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -82,12 +82,20 @@ To view bug fixes, see the [GitHub milestone for 2201.4.0 (Swan Lake)](https://g
 ## Developer tools updates
 
 ### New features
+#### OpenAPI Tool
+Added Swagger version 2 support for OpenAPI to Ballerina client, service generation.
 
 #### Language Server
 
 ### Improvements
 
 #### OpenAPI Tool
+- Added mapping for the `additionalProperty` property in the OpenAPI object schema. With this support, generated Ballerina records can be open records or closed records according to the `additionalProperty` property details.
```

Comment:
@lnash94 What's the correct property name in here? is it `additionalProperty` or `additionalProperties`?

Reply:
yeah it should be a `additionalProperties` . will change